plugins/star-rating/frontend/public/stylesheets/countly-feedback.css

```diff
@@ -0,0 +1,525 @@
+.rating-emotion {
+     width: 67.5px;
+     height: 67.5px;
+     padding: 5px;
+}
+/*4mobile*/
+@media screen and (max-device-width: 414px) {
+    /* Modal Content/Box */
+     .modal-content {
+         background-color: #fefefe;
+         position: relative;
+         box-sizing: border-box;
+         width: 90%;
+         margin-left: 5%;
+         box-sizing: border-box;
+         padding-bottom: 25px;
+         padding-top: 25px;
+         border-radius: 5px;
+    }
+    /* Modal Content/Box */
+     .success-modal-content {
+         position: relative;
+         background-color: #fefefe;
+         box-sizing: border-box;
+         width: 90%;
+         margin-left: 5%;
+         padding-bottom: 25px;
+         padding-top: 25px;
+         border-radius: 5px;
+         display: none;
+    }
+     #question-area {
+        font-family: 'Lato', sans-serif;
```

Comment:
I see font files being add and fonts being user, but I don't see any font-face definitions for the font files, like we have here, as example:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/stylesheets/main.css#L13